qiskit/algorithms/time_evolvers/trotterization/trotter_qrte.py

```diff
@@ -55,16 +59,23 @@ def __init__(
         self,
         product_formula: ProductFormula | None = None,
         estimator: BaseEstimator | None = None,
+        num_timesteps: int = 1,
     ) -> None:
         """
         Args:
             product_formula: A Lie-Trotter-Suzuki product formula. If ``None`` provided, the
-                Lie-Trotter first order product formula with a single repetition is used.
+                Lie-Trotter first order product formula with a single repetition is used. ``reps``
+                should be 1 to obtain a number of time-steps equal to ``num_timesteps`` and an
+                evaluation of ``TimeEvolutionProblem.aux_operators`` at every time-step. If ``reps``
```

Comment:
We could link to this 🙂 
```suggestion
                evaluation of :attr:`.TimeEvolutionProblem.aux_operators` at every time-step. If ``reps``
```